clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -41,3 +41,37 @@
       In: ${arg}.cu
       Out: ${arg}.dp.cpp
 
+- Rule: rule_cuda_add_cufft_to_target
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: cuda_add_cufft_to_target
+  MatchMode: Partial
+  In: cuda_add_cufft_to_target(${value})
+  Out: target_link_libraries(${value} PUBLIC -L${MKLROOT}/lib/intel64 -lmkl_sycl -lmkl_intel_ilp64 -lmkl_tbb_thread -lmkl_core)
+
+- Rule: rule_cxx_target_compile_features
```

Comment:
Pls add comment for this rule like:
Current Yaml based rule hard-code to map "cxx_std_xx" to "cxx_std_17", more flexible mapping logic should be implemented in implicit rules.